.gitlab-ci.yml

```diff
@@ -0,0 +1,96 @@
+image: docker:stable
```

Comment:
I'm not sure if I understand why you added this `.gitlab-ci.yml` file - despite the explanation in https://github.com/benmarwick/rrtools/issues/91#issuecomment-544291498.

I mean: This would be excellent to have if the rrtools package was hosted on gitlab. Then we would have a nice testing infrastructure. But it's currently not. So our .travis.yml is fine, this file we do nothing and we would have to maintain it without actually using it.

Or am I missing something?